deisctl/cmd/cmd.go

```diff
@@ -179,7 +179,7 @@ func startDefaultServices(b backend.Backend, stateless bool, wg *sync.WaitGroup,
 	b.Start([]string{"publisher"}, wg, out, err)
 	wg.Wait()
 
-	fmt.Fprintln(out, "Routing mesh...")
+	fmt.Fprintln(out, "Router mesh...")
```

Comment:
This change brings the output in-line with the nomenclature used in the docs, where we always use "router mesh" and not "routing mesh."